generator_hdl.py

```diff
@@ -355,12 +366,9 @@ def writeTestBench(tbfunc, topfunc, process_list, memDict, memInfoDict, memPrint
     string_header += writeTBOpener(tbfunc)
 
     string_constants = writeTBConstants(memDict, memInfoDict, notfinal_procs+[final_procs[-1].mtype_short()], memPrintsDir, sector, split)
-    # A bodge for TrackBuilder to write TF concatenated track+stub data.
-    # (Needed to compare with emData/).
-    if 'TW_104' in memInfoDict.keys():
-      fileTF = open("bodge/TF_tb_constants.vhd.bodge")
-      string_constants += fileTF.read();
-
+    if 'TW_113' in memInfoDict.keys():
```

Comment:
@aryd The magic 113 can be removed with something like the following:
```suggestion
    if len([key for key in memInfoDict if key.startswith('TW_')]) > 0:
```

Reply:
Fixed